modules/ocm-accesscontrol-tab.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// ocm/ocm-overview.adoc
+
+[id="ocm-accesscontrol-tab_{context}"]
+= Access control tab
+
+This tab allows the cluster owner to grant roles to other users.
+
+.Prerequisites
+
+. You must be the cluster owner or have the correct permissions to grant roles on the cluster.
+
+.Procedure
```

Comment:
This tab is not only for role grant in oCM. It also support IDP configuration and aws infrastructure access grant. Is it enough to just introduce the OCM role grant?